### ISSUE TITLE
I want to increase learning_rate and train_workers, is that possible? 

### ISSUE THREAD

**Issue by DucBac99** [pythonlessons/mltu#35]
@pythonlessons please help me

**Comment by pythonlessons** [comment]
Which tutorial? Simply change these values and you good to go

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/8a9b4de0-fe16-4217-997c-937451ea57ab)
I want to change the values in the config file to increase training speed, is it possible? @pythonlessons

**Comment by pythonlessons** [comment]
it really depends on your machine, if your GPU is used 100% batch size and workers number will not help, then you need to change model architecture or input size or other stuff

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/54cd6e0b-155c-4004-b8c7-d717fd37bfe0)
This is my computer configuration, I find the train is very slow, and it can only be turned off after more than 100 trains.

**Comment by pythonlessons** [comment]
So, you are training model on CPU. That's why it takes a lot of time. You should train it on GPU, if you don't have one try to use google collab and try training on free GPU

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/12ea8d9d-d219-4c5f-9250-085be5be145a)
I'm also configuring training using GPU like you. I also tried using google collab but I found it too slow @pythonlessons

**Comment by pythonlessons** [comment]
I am configuring only ram, not reserving all possible GPU ram, you need to make sure your system can see GPU device (install drivers, CUDA, cuDNN).

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/3a012369-ddbc-4879-89e5-b2688b0771fc)
I can see the GPU but when I run the train it still only runs through the CPU

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/f439f41a-6122-401b-9238-30987f464851)
I noticed that if I trained more than 100 times, I would get this error. I also tried searching the val.csv file in the model and only got 1726 records while my dataset had more than 20 thousand images.
![image](https://github.com/pythonlessons/mltu/assets/77089341/f2952ea0-207d-43c5-9a3a-50c69ff5b2e5)

**Comment by pythonlessons** [comment]
You received this error, because you try to convert model to onnx, but your GPU is too old to support it. You need to convert it manually using cpu onnx version. Your 20k images dataset should be split into train and validation, check how much data you had in your train.csv

**Comment by DucBac99** [comment]
> You received this error, because you try to convert model to onnx, but your GPU is too old to support it. You need to convert it manually using cpu onnx version. Your 20k images dataset should be split into train and validation, check how much data you had in your train.csv

I checked that the train runs entirely on CPU and not on GPU. I tried print("Num GPUs Available: ", len(tf.config.list_physical_devices('GPU'))) returned = 0, is there any way to fix it? fix this error k

**Comment by DucBac99** [comment]
![image](https://github.com/pythonlessons/mltu/assets/77089341/b24f90c7-cb03-41fe-aaa2-ccc52daac35b)
this is the data in my train.csv

**Comment by pythonlessons** [comment]
solved?

**Comment by pythonlessons** [comment]
Early stopping is not an error, its a function, if you model is not learning enough, increase early stopping patience, try playing around with learning rate or even model. Maybe you images are hard to crack

**Comment by DucBac99** [comment]
> Early stopping is not an error, its a function, if you model is not learning enough, increase early stopping patience, try playing around with learning rate or even model. Maybe you images are hard to crack

My captcha has 6 characters, do I need to change the model?

**Comment by DucBac99** [comment]
> no, you dont

Can you solve the image pulling captcha and image rotation captcha
